packages/strapi-plugin-content-manager/test/index.test.e2e.js

```diff
@@ -351,6 +351,29 @@ describe('Content Manager End to End', () => {
       expect(body.message).toBe('Already published');
     });
 
+    test('Unpublish article1, expect article1 to be set to null', async () => {
+      const entry = _.clone(data.articles[0]);
```

Comment:
Why do you clone? Can we remove it? Same for  the 2 tests above and the one below

Reply:
We can :)